src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -73,6 +86,12 @@ export const GettingStarted = () => {
   const paddingCss = css`
     padding: calc(${euiTheme.size.base}*3) calc(${euiTheme.size.base}*4);
   `;
+
+  const isDarkTheme = uiSettings.get<boolean>('theme:darkMode');
+  const activateGuide = async (useCase: UseCase, guideState?: GuideState) => {
+    await guidedOnboardingService?.activateGuide(useCase as GuideId, guideState);
+    // TODO error handling
```

Comment:
We don't have the correct error handling in the API yet, so this will be addressed in a separate [issue](https://github.com/elastic/kibana/issues/139798)

Reply:
Could you link the issue in the TODO then?